.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
[OPTIONAL] I recommend you remove this file from your repo and add it to your `.gitignore` file. Sometimes your IDE and local environment will create files inside your project that are not related to it. This will help you keep your repo clean and professional.